lib/app/features/feed/stories/views/components/story_preview/media/story_image_preview.dart

```diff
@@ -20,28 +17,11 @@ class StoryImagePreview extends StatelessWidget {
   Widget build(BuildContext context) {
     return ClipRRect(
       borderRadius: BorderRadius.circular(16.0.s),
-      child: Consumer(
-        builder: (context, ref, _) {
-          final assetEntityAsync = ref.watch(assetEntityProvider(path));
-
-          return assetEntityAsync.maybeWhen(
-            data: (asset) {
-              if (asset == null) {
-                return const CenteredLoadingIndicator();
-              }
-
-              return Image(
-                image: AssetEntityImageProvider(
-                  asset,
-                  isOriginal: false,
-                  thumbnailSize: const ThumbnailSize.square(600),
-                ),
-                fit: BoxFit.cover,
-              );
-            },
-            orElse: () => const CenteredLoadingIndicator(),
-          );
-        },
+      child: Image.file(
+        File(path),
+        fit: BoxFit.cover,
+        width: 600.0.s,
+        height: 600.0.s,
```

Comment:
@ice-alcides does it break your Story logic?

Reply:
It looks good, but @ice-kreios, could you please check that it works in both cases: selecting media from a gallery and capturing from the camera?

Reply:
and adding `loadingBuilder` (https://api.flutter.dev/flutter/widgets/Image/loadingBuilder.html) might be added for better UX